pkg/handler/handler.go

```diff
@@ -183,9 +184,23 @@ func (h handler) bind(w http.ResponseWriter, r *http.Request, params map[string]
 		return
 	}
 
+	// process binding request
 	resp, err := h.broker.Bind(instanceUUID, bindingUUID, req)
 
-	writeDefaultResponse(w, http.StatusCreated, resp, err)
+	if err != nil {
+		switch err {
+		case broker.ErrorDuplicate:
+			writeResponse(w, http.StatusConflict, broker.BindResponse{})
+		case broker.ErrorAlreadyProvisioned:
```

Comment:
That's a hard one because there's no good way to returned what seems like an ok state since there are 3 good states, OK, ACCEPTED, & CREATED. So you *could* look at it as a good case is we go to create one, and trying to provision the SAME instance is an error case since we already did that. The 200 OK is per the spec. That's the thought process I went with since it is an exceptional case of provision.